app/components/Modal/HtlcModal.jsx

```diff
@@ -968,8 +825,8 @@ class HtlcModal extends React.Component {
             operation && operation.type === "create"
                 ? counterpart.translate("showcases.htlc.create_htlc")
                 : isExtend
-                    ? counterpart.translate("showcases.htlc.extend_htlc")
-                    : counterpart.translate("showcases.htlc.redeem_htlc");
+                ? counterpart.translate("showcases.htlc.extend_htlc")
```

Comment:
This refactoring here is not reproduced by my prettifier hook. Are you sure you don't have a local format on save or something? It's really weird

Reply:
really weird, agreed. Maybe VScode and Webshtorm have different default settings?